nlu/universe/component_universes.py

```diff
@@ -2647,6 +2651,25 @@ class ComponentUniverse:
                                         jsl_anno_py_class=ACR.JSL_anno_HC_ref_2_py_class[H_A.MEDICAL_SUMMARIZER],
                                         ),
 
+        A.BART_TRANSFORMER: partial(NluComponent,
+                                    name=A.BART_TRANSFORMER,
+                                    type=T.DOCUMENT_CLASSIFIER,
+                                    get_default_model=SparkNLPBartTransformer.get_default_model,
+                                    get_pretrained_model=SparkNLPBartTransformer.get_pretrained_model,
+                                    pdf_extractor_methods={'default': default_T5_config,
+                                                           'default_full': default_full_config, },
+                                    pdf_col_name_substitutor=substitute_summarizer_cols,
+                                    output_level=L.INPUT_DEPENDENT_DOCUMENT_CLASSIFIER,
+                                    node=NLP_FEATURE_NODES.nodes[A.BART_TRANSFORMER],
```

Comment:
`substitute_summarizer_cols` is not a good choice. COlumn names will have `summary` , but bart is a text generator.   Makes more sense to use `substitute_gpt2_cols` for generation here and also for the pdf_extractor